lib/membrane_overlay_filter.ex

```diff
@@ -100,9 +100,12 @@ defmodule Membrane.OverlayFilter do
   end
 
   defp compose_planes({image_y, image_u, image_v}, {overlay_y, overlay_u, overlay_v}, opts) do
+    uv_x = if is_integer(opts[:x]), do: div(opts[:x], 2), else: opts[:x]
+    uv_y = if is_integer(opts[:y]), do: div(opts[:y], 2), else: opts[:y]
+    opts_uv = [x: uv_x, y: uv_y]
```

Comment:
- you forgot blend_mode
- maybe it would be better to do this in `handle_init` and keep opts as a tuple `{opts_y, opts_uv}`